functions.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+function legoCollection (array $legoSets) : string {
+
+    if($legoSets === []){
+        $displayLego = 'There is an error with the database, please try again later.';
```

Comment:
error message might not be database, this could happen if there are no lego sets. Best to say 'I cant find any lego sets at the moment, please try again later.'